src/alacris_lib/operator_contract.ml

```diff
@@ -23,13 +27,36 @@ let topic_of_amount (amnt : TokenAmount.t) : Bytes.t option =
 let topic_of_hash (hash : Digest.t) : Bytes.t option =
   Some (encode_function_parameters [abi_digest hash])
 
+(* Topics below correspond to events in the operator.sol code
+   Be careful of adjusting everything when you change the type like adding a balance.
+ *)
+
+let (topic_of_deposited: Bytes.t option) =
+  topic_of_hash (digest_of_string "Deposited(address,address,uint256,uint256)")
```

Comment:
maybe define, export and use this library function:
`let topic_of_signature = digest_of_string >> topic_of_hash`

Reply:
Ok, done